src/sentry/replays/blueprints/api.md

```diff
@@ -488,6 +488,51 @@ With download query argument.
 
   Content-Type application/octet-stream
 
+## Replay Videos [/projects/<organization_slug>/<project_slug>/replays/<replay_id>/videos/]
+
+- Parameters
+  - per_page
+  - cursor
+
+### Browse Replay Videos [GET]
+
+Retrieve a collection of replay videos.
+
+| Column     | Type   | Description |
+| ---------- | ------ | ----------- |
+| segment_id | number | -           |
+| url        | string | -           |
+
+- Response 200
+
+  ```json
+  {
+    "data": [
+      {
+        "segment_id": 0,
+        "url": "https://storage.google.com/presigned-url?file=3151351351"
```

Comment:
can we include timestamp + duration here?

Reply:
Video metadata is in the RRWeb JSON.